nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/DpvSettings.java

```diff
@@ -0,0 +1,12 @@
+package no.difi.meldingsutveksling.nextmove;
+
+import lombok.Data;
+
+@Data
+public class DpvSettings {
+    private DpvVarselType varselType;
```

Comment:
Flisespikkeri.. men burde denne heite DpvVarslingType og DpvVarslingTransportType etc i staden for varsel ? Synes det høyres meir rett ut

Reply:
Gikk for "Varsel" siden det gjelder dette konkrete varselet, og ikke "varsling" generelt. Det var tankegangen, i alle fall :)